src/sources/splunk_hec.rs

```diff
@@ -398,8 +398,12 @@ impl<R: Read> Stream for EventStream<R> {
                     if let Some(line) = object.remove("line") {
                         match line {
                             // This don't quite fit the meaning of a event::schema().message_key
-                            JsonValue::Array(_) | JsonValue::Object(_) => log.insert("line", line),
-                            _ => log.insert(event::log_schema().message_key(), line),
+                            JsonValue::Array(_) | JsonValue::Object(_) => {
+                                log.insert("line", line);
+                            }
+                            _ => {
+                                log.insert(event::log_schema().message_key(), line);
+                            }
```

Comment:
Just a matter of style, but it seems like the number of lines of code could have been reduced by putting `;` after the `match` statement rather than after each of the branches.

Reply:
You're right!